WalletWasabi.Fluent/ViewModels/Wallets/Buy/Messages/PayNowAssistantMessageViewModel.cs

```diff
@@ -86,7 +90,10 @@ private async Task SendAsync(Wallet wallet, TransactionInfo info)
 
 				await Services.TransactionBroadcaster.SendTransactionAsync(transaction.Transaction);
 				wallet.UpdateUsedHdPubKeysLabels(transaction.HdPubKeysWithNewLabels);
-				IsPaid = true;
+
+				var updatedMessage = Message with { Data = Invoice with { IsPaid = true } };
+				var updatedConversation = _workflow.Conversation.ReplaceMessage(Message, updatedMessage);
+				_workflow.Conversation = updatedConversation;
```

Comment:
Conversation must be saved here, otherwise this information might get lost.

use `buyAnythingManager.UpdateConversationAsync()`

Reply:
Discussed on Slack that there is is auto save logic, which we will remove later, but for now it does the job.